WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/logic/document/DocumentCreator.java

```diff
@@ -51,16 +55,22 @@ private static String getEditorOrFileContent(ITextEditor editor) {
 		} catch (IllegalArgumentException | ContentReaderException exception) {
 			// Editor was null, there is nothing we can do to get the file
 			// contents.
-			WatchDogLogger.getInstance().logSevere(exception);
+			WatchDogLogger.getInstance(

```

Comment:
what is happening here? Why is WatchDogGlobals.getPreferences().isLoggingEnabled() parameter to getInstance of logger?